custom_components/localtuya/cover.py

```diff
@@ -112,8 +112,11 @@ def __init__(
         switchid,
         **kwargs,
     ):
-        #_LOGGER.info("running def __init__ of LocaltuyaCover(CoverEntity) with self=%s device=%s name=%s friendly_name=%s icon=%s switchid=%s open_cmd=%s close_cmd=%s stop_cmd=%s", self, device, name, friendly_name, icon, switchid, open_cmd, close_cmd, stop_cmd)
         super().__init__(device, config_entry, switchid, **kwargs)
+        self._config[CONF_OPEN_CMD] = "on"
```

Comment:
What is happening here? Why are you assigning these values here? They should be default from the schema. Otherwise we should just use whatever value they have.

Reply:
I told you, it's because I could not make the Options schema work ...

Reply:
Right, I assume you are referring to the `_convert_entity` stuff? Should hopefully work with if you do as I wrote. Otherwise you can push the changes you have so I can try it out myself.